webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -112,11 +112,13 @@ export default {
       this.showCropper = false
       const canvas = this.cropper.getCroppedCanvas()
       canvas.toBlob(blob => {
+        const imageAspectRatio = canvas.width / canvas.height
         this.setupPreview(canvas)
         this.removeCropper()
-        const croppedImageFile = new File([blob], this.file.name, { type: 'image/jpeg' })
+        const croppedImageFile = new File([blob], this.file.name, { type: this.file.type })
```

Comment:
Does that allow us to save SVG files, too? How is it possible that it didn't complain about GIF or PNG files?

Reply:
I think it would, although there was an unintended removal of `image/jpeg` from line 119
this change would allow us to support `svg`, which is certainly what we want, but would give remove our compression.
I'll add that back and we can work on a solution that compresses in the backend and supports `svg` files